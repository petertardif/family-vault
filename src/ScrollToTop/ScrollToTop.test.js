import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ScrollToTop from './ScrollToTop'

describe(`ScrollToTop component`, () => {

  it('renders a ScrollToTop by default', () => {
    const wrapper = shallow(<ScrollToTop />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})