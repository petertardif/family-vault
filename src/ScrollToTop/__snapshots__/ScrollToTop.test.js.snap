// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop component renders a ScrollToTop by default 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
